pkgs/by-name/br/breakpointHook/attach.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash
+
+id="$1"
+pid="$(pgrep -f "sleep $id" | head -n1)"
+
+# helper to extract variables from the build env
+getVar(){
+  nsenter --mount --target "$pid" /bin/sh -c "
```

Comment:
I would replace this with `xargs -0 < /proc/$pid/environ`.
Reason is that some environment variables might be set after env-vars and those might be important for the build as well.

Reply:
Rather than xargs, I think `read` might be better actually. It will also work if environment variables contain newline. Something like this: https://stackoverflow.com/a/8677566 minus reading from a file instead of a process

Reply:
Ok. This is actually not used for all environment variables...

Reply:
It would be more sound if we could use environ instead of `/build/env-vars`